test/e2e/rte/rte.go

```diff
@@ -130,7 +139,7 @@ var _ = ginkgo.Describe("[RTE][InfraConsuming] Resource topology exporter", func
 					return false
 				}
 				return reason == nrtupdater.RTEUpdateReactive
-			}, 5*time.Second, 1*time.Second).Should(gomega.BeTrue(), "didn't get updated node topology info")
```

Comment:
This ```timeout``` value is not a multiple of ```poll interval``` value (the default - and in our case we're using the default is ```10s```) so I'm not sure I understand why the new implementation is better.
What am I missing here?

Reply:
I missed to update this part :)